### PR TITLE
[wrangler] deployment command

### PR DESCRIPTION
**Pull request by JacobMGEvans**
Add documentation reference in command section of Wrangler for new  command feature.

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Wrangler offers a number of commands to manage your Cloudflare Workers.
 - [`login`](#login) - Authorize Wrangler with your Cloudflare account using OAuth.
 - [`logout`](#logout) - Remove Wrangler’s authorization for accessing your account.
 - [`whoami`](#whoami) - Retrieve your user information and test your authentication configuration.
+- [`deployments`](#deployments) - Logs the 10 most recent deployments with 'Version ID', 'Version number','Author name', and 'Latest deploy'.
```
Comment (lrapoport-cf) [inline content/workers/wrangler/commands.md]: to be consistent with the tense used in defining the other commands, suggest using the imperative mood. and to be consistent with how `whoami` is defined, suggest changing from "log" to "retrieve". also suggest we leave the info about what is returned in the detail section for the command

```suggestion
- [`deployments`](#deployments) - Retrieve details for the 10 most recent deployments.
```

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -1185,3 +1186,17 @@ Retrieve your user information and test your authentication configuration.
...
+
+Logs the 10 most recent deployments with 'Version ID', 'Version number','Author name', and 'Latest deploy'
+
+```sh
+$ wrangler deployments
+```
+
+{{<definitions>}}
+
+- `--name` {{<type>}}string{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (lrapoport-cf) [inline content/workers/wrangler/commands.md:1202]: what does this flag do? we should define

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -1185,3 +1186,17 @@ Retrieve your user information and test your authentication configuration.
 ```sh
 $ wrangler whoami
 ```
+
+## deployments
+
+Logs the 10 most recent deployments with 'Version ID', 'Version number','Author name', and 'Latest deploy'
```
Comment (lrapoport-cf) [inline content/workers/wrangler/commands.md]:
```suggestion
Retrieve details for the 10 most recent deployments. Details include `Version ID`, `Version number`, `Author name`, and `Latest deploy`.
```

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (JacobMGEvans) [reply content/workers/wrangler/commands.md:1202]: Done. Its allows you to specify a script name rather than Wrangler inferring it.

**Review by lrapoport-cf — APPROVED** [review]
approved but should not be merged until https://github.com/cloudflare/wrangler2/pull/2091 is merged

File: content/workers/wrangler/commands.md [hunk 4]
```diff
@@ -1185,3 +1186,18 @@ Retrieve your user information and test your authentication configuration.
 ```sh
 $ wrangler whoami
 ```
+
+## deployments
+
+Retrieve details for the 10 most recent deployments. Details include `Version ID`, `Version number`, `Author name`, and `Latest deploy`.
```
Comment (lrapoport-cf) [inline content/workers/wrangler/commands.md]: per https://github.com/cloudflare/wrangler2/pull/2091#discussion_r1009533087 and https://github.com/cloudflare/wrangler2/pull/2091#discussion_r1009557379: 

```suggestion
Retrieve details for the 10 most recent deployments. Details include `Version ID`, `Version number`, `Author email`, `Created on`, and `Latest deploy`.
```

**Review by lrapoport-cf — CHANGES_REQUESTED** [review]

File: content/workers/wrangler/commands.md (hunk 4 above)
Comment (JacobMGEvans) [reply content/workers/wrangler/commands.md:1192]: Fixes on https://github.com/cloudflare/wrangler2/pull/2108

Comment (JacobMGEvans) [reply content/workers/wrangler/commands.md]: I thought this was a different PR lol

**Review by lrapoport-cf — APPROVED** [review]

**Pull request merged**